src/core_ops.rs

```diff
@@ -60,7 +60,28 @@ pub fn op_listp(a: &mut Allocator, input: NodePtr, _max_cost: Cost) -> Response
 
 pub fn op_raise(a: &mut Allocator, input: NodePtr, _max_cost: Cost) -> Response {
     let args = Node::new(a, input);
-    args.err("clvm raise")
+    // Quexington requested single argument raise to return the single argument
+    // rather than the full list of arguments.  brun also used to behave this
+    // way... it does make sense to ensure that the full context of the exception
+    // is passed on rather than a more cryptic error in the case of more arguments
+    // though (IMO).  We restrict the single argument case to atoms since it
+    // ensures that a single argument with a pair is distinguishable from multiple
+    // independent arguments (as new, re: richard).
```

Comment:
This might be good history, but I'd prefer if the comment just explained what it the code is doing.

Reply:
ok i think all those are done